<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="r_btn_closet">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>40</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
background-color: rgb(177, 236, 240);
border:0px;</string>
   </property>
   <property name="text">
    <string>옷장</string>
   </property>
  </widget>
  <widget class="QPushButton" name="r_btn_main">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
background-color: rgb(177, 236, 240);
border:0px;</string>
   </property>
   <property name="text">
    <string>메인</string>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_year">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_day">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_state">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_img_clothes1">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>150</y>
     <width>361</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes1_n">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>630</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_img_clothes2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>180</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_img_clothes3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>430</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes1_s">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>700</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes1_t">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>700</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes2_t">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>290</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes2_n">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>220</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes2_s">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>290</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes3_t">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>540</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes3_s">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>540</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes3_n">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>470</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_tem">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 216, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
